src/main/kotlin/uk/gov/justice/digital/hmpps/hmppsaccreditedprogrammesapi/client/oasysApi/model/OasysRoshSummary.kt

```diff
@@ -1,16 +1,51 @@
 package uk.gov.justice.digital.hmpps.hmppsaccreditedprogrammesapi.client.oasysApi.model
 
+import com.fasterxml.jackson.annotation.JsonCreator
 import com.fasterxml.jackson.annotation.JsonIgnoreProperties
+import com.fasterxml.jackson.annotation.JsonValue
 
 @JsonIgnoreProperties(ignoreUnknown = true)
 data class OasysRoshSummary(
-  val riskPrisonersCustody: String?,
-  val riskStaffCustody: String?,
-  val riskStaffCommunity: String?,
-  val riskKnownAdultCustody: String?,
-  val riskKnownAdultCommunity: String?,
-  val riskPublicCustody: String?,
-  val riskPublicCommunity: String?,
-  val riskChildrenCustody: String?,
-  val riskChildrenCommunity: String?,
+  val riskPrisonersCustody: ScoreLevel?,
```

Comment:
ScoreLevel - do we need this object? oasys has it as a string. so if they change anything at their end , it could potentially break our code